packages/client-api-schema/src/methods/CloseAndWithdraw.ts

```diff
@@ -1,8 +1,7 @@
-import {Participant} from '../data-types';
 import {JsonRpcRequest, JsonRpcResponse, JsonRpcError} from '../utils';
 import {ErrorCodes} from '../error-codes';
 
-export type CloseAndWithdrawParams = {hub: Participant; playerParticipantId: string};
+export type CloseAndWithdrawParams = {hubParticipantId: string};
```

Comment:
Here's the change to the JSON-RPC schema. It brings it into line with the channel client API. Advantage being, the channel client does not have to fill in the remaining fields as the request comes across. 

It also brings this schema entry close to the `getBudget` params (but with the remaining difference being `hubAddress` vs `hubParticipantId`. We may want to bring those in line with each other.